services-camel/src/test/java/edu/unc/lib/dl/services/camel/images/AddDerivativeProcessorTest.java

```diff
@@ -81,69 +73,43 @@
     @Before
     public void init() throws Exception {
         initMocks(this);
-        processor = new AddDerivativeProcessor(repoObjLoader, slug, fileExtension, mimetype, maxRetries, retryDelay);
-        file = File.createTempFile(fileName, ".PNG");
-        file.deleteOnExit();
-        when(exchange.getIn()).thenReturn(message);
 
-        when(repoObjLoader.getBinaryObject(any(PID.class))).thenReturn(binary);
+        TestHelper.setContentBase(FEDORA_BASE);
+
+        finalDir = moveDir.getRoot();
+
+        processor = new AddDerivativeProcessor(fileExtension, finalDir.getAbsolutePath());
+
+        pathId = PIDs.get(RESC_ID).getId();
+
+        file = tmpDir.newFile(pathId);
+
+        when(exchange.getIn()).thenReturn(message);
 
         when(message.getHeader(eq(FCREPO_URI)))
```

Comment:
This should be RESC_ID instead of what's happening on line 92

Reply:
looks like this was fixed, no?